crates/cargo-rm/src/cargo/ops/mod.rs

```diff
@@ -0,0 +1,3 @@
+//! Core of cargo commands
+
+pub mod cargo_rm;
```

Comment:
I expect the command will be `remove` with a `rm` alias.

Updating everything to reflect that can happen in a later PR which is why I went ahead and merged.